.rubocop.yml

```diff
@@ -0,0 +1,78 @@
+AllCops:
+  TargetRubyVersion: 2.5.1
+  DisplayCopNames: true
+  Exclude:
+    - db/schema.rb
+    - db/migrate/**/*
+    - kithe.gemspec
+    - Rakefile
+    - Gemfile
+    - bin/rails
+    - spec/**/*
+    - test/**/*
+    - config/**/*
+    - lib/**/*
+    # - app/**/*
+
+
+Metrics/LineLength:
+  Max: 110
+
+Layout:
+  EnforcedStyle: rails
+
+Style/CollectionMethods:
+  PreferredMethods:
+    collect: 'map'
+    collect!: 'map!'
+    inject: 'reduce'
+    detect: 'find'
+    find_all: 'select'
```

Comment:
i always use `collect` and `find_all`. Maybe I should finally join everyone else?